src/main/java/org/sbml/spatial/segmentation/crossVal.java

```diff
@@ -34,13 +34,14 @@ public static void main(String[] args) {
 		try {
 			
             int batchSize = 1;
-            String home = System.getProperty("user.home");
-            System.out.println(home);
-            
+            String directory = System.getProperty("user.dir");
+
+            //String home = System.getProperty("user.dir");
+
             DataNormalization scaler = new ImagePreProcessingScaler(); // scale image between 0 and 1
             UnetPathLabelGenerator labeler = new UnetPathLabelGenerator();
-            //File rootDir = new File("C:\\Users\\Subroto\\small_dataset");
-            File rootDir = new File(home + File.separator + "small_dataset");
+            //File rootDir = new File("C:\\Users\\Subroto\\unetdl4j\\dataset\\small_dataset");
```

Comment:
Probably remove this line since we won't be needing it

Reply:
Just kept it as an indicator for myself. Will remove it.